src/DIRAC/Core/scripts/dirac_apptainer_exec.py

```diff
@@ -71,7 +71,6 @@ def main():
 
     # Now let's construct the apptainer command
     cmd = ["apptainer", "exec"]
-    cmd.extend(["--contain"])  # use minimal /dev and empty other directories (e.g. /tmp and $HOME)
```

Comment:
I'm pretty sure this is breaking half of the security gaurentee we get from SingularityCE. If this isn't the case we at the very least need a clear explanation in the description of why it isn't.

I also don't think it even fixes the error in the description.